services/keyboard/src/org/chromium/mojo/keyboard/KeyboardServiceImpl.java

```diff
@@ -17,20 +17,12 @@
  * Android implementation of Keyboard.
  */
 public class KeyboardServiceImpl implements KeyboardService {
```

Comment:
Well, hopefully they won't diverge for very long, the Mojo implementation is up next :)